bundles/automation/org.eclipse.smarthome.automation.json.parsers/.project

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.eclipse.smarthome.automation.json.parsers</name>
```

Comment:
I think it would be better to reverse the package naming to org.eclipse.smarthome.automation.parser.json, as then we can also add an alternative org.eclipse.smarthome.automation.parser.gson.